src/main/webapp/view/transcript/transcript.js

```diff
@@ -85,6 +86,11 @@ function appendTextToList(transcriptLine, ulElement) {
   ulElement.appendChild(liElement);
   // Save the dates for seeking later.
   liElement.startDate = new Date(transcriptLine.start);
+  liElement.endDate = new Date(transcriptLine.end);
```

Comment:
Why do we need a date here?

Reply:
The start date is stored so that when the Transcript calls `seekAll`, it has a time to pass into the parameter. The end date is used to check that the time received from the video is still within the current transcript line.